components/director/internal/tenantmapping/handler.go

```diff
@@ -105,3 +109,105 @@ func (h *Handler) setScopes(extraMap map[string]interface{}) {
 
 	extraMap["scope"] = strings.Join(scopes, " ")
 }
+
```

Comment:
Please write missing unit tests for not covered test cases:

![Screen Shot 2019-10-11 at 12 39 50](https://user-images.githubusercontent.com/7155799/66645650-51fca380-ec24-11e9-8d78-4465be1372e8.png)